### PR TITLE
S01 us31

### PR DESCRIPTION
**Pull request by Kino-ki**
"refractor: database done & map and navbar created,"

(take 2) : corrections made.
previsous pull request closed

### CODE REVIEW

**Review by Jhanemba64 — APPROVED** [review]

**Review by Jubdc — APPROVED** [review]

File: frontend/src/components/Game/Map/LeafletMap.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <TileLayer
+          attribution='&copy; <a href="https://stadiamaps.com/">Stadia Maps</a> contributors'
+          url="https://tiles.stadiamaps.com/tiles/stamen_watercolor/{z}/{x}/{y}.jpg"
+        />
+        <TileLayer
+          attribution='&copy; <a href="https://stadiamaps.com/">Stadia Maps</a> contributors'
+          url="https://tiles.stadiamaps.com/tiles/stamen_watercolor/{z}/{x}/{y}.jpg"
+        />
+        <Marker position={[45.51683, -73.58283]} icon={PinIcon}>
+          {/* //faire un component pour le popup et son contenu */}
```
Comment (ioayoub) [inline frontend/src/components/Game/Map/LeafletMap.jsx:31]: Remove this comment

File: frontend/src/components/NavBar.jsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    {
+      path: "/game/ranks",
+      title: "GALERIE /SCORES",
+    },
+    {
+      path: "/administration/main",
+      title: "ADMINISTRATION",
+    },
+  ];
+  // ----------------------- [for styling purpose] useState to store the page "name" we're in---------------------
```
Comment (ioayoub) [inline frontend/src/components/NavBar.jsx:28]: Remove this

File: frontend/src/components/NavBar.jsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  const [page, setPagae] = useState(null);
+  // -----------------[for styling purpose] function to ref the page name and put it in the state --------------
```
Comment (ioayoub) [inline frontend/src/components/NavBar.jsx:30]: Remove this

File: frontend/src/components/NavBar.jsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  };
+
+  return (
+    <div>
+      <nav className="flex flex-row mr-4 text-xl font-semibold text-primary_color ">
+        <NavLink to="/home">
+          <img alt="logo" src={steve} className="ml-5 hover:animate-spin" />
+        </NavLink>
+        <ul className="flex flex-row justify-end gap-4 ml-64 text-primary">
+          {/* [for styling purpose] conditional rendereing depending on page state (page name takes a diferent padding when page displayed) */}
```
Comment (ioayoub) [inline frontend/src/components/NavBar.jsx:42]: Remove this

**Review by Halicksse — APPROVED** [review]

File: frontend/src/pages/Administration/AdminPage.jsx [hunk 5]
```diff
@@ -0,0 +1,7 @@
+export default function ArtworksListPage() {
```
Comment (ioayoub) [inline frontend/src/pages/Administration/AdminPage.jsx:1]: Empty pages are not allowed

File: frontend/src/pages/Administration/ArtworksListPage.jsx [hunk 6]
```diff
@@ -0,0 +1,3 @@
+export default function ArtworksListPage() {
```
Comment (ioayoub) [inline frontend/src/pages/Administration/ArtworksListPage.jsx:1]: Empty pages are not allowed

File: frontend/src/pages/Administration/RequestValidationPage.jsx [hunk 7]
```diff
@@ -0,0 +1,3 @@
+export default function RequestValidationPage() {
```
Comment (ioayoub) [inline frontend/src/pages/Administration/RequestValidationPage.jsx:1]: Empty pages are not allowed

File: frontend/src/pages/Administration/UserListPage.jsx [hunk 8]
```diff
@@ -0,0 +1,3 @@
+export default function UserListPage() {
```
Comment (ioayoub) [inline frontend/src/pages/Administration/UserListPage.jsx:1]: Empty pages are not allowed

File: frontend/src/pages/Game/RanksPage.jsx [hunk 9]
```diff
@@ -0,0 +1,7 @@
+export default function RanksPage() {
```
Comment (ioayoub) [inline frontend/src/pages/Game/RanksPage.jsx:1]: Empty pages are not allowed

File: frontend/src/pages/Game/SendPicturePage.jsx [hunk 10]
```diff
@@ -0,0 +1,3 @@
+export default function SendPicturePage() {
```
Comment (ioayoub) [inline frontend/src/pages/Game/SendPicturePage.jsx:1]: Empty pages are not allowed

File: frontend/src/pages/User/LoginPage.jsx [hunk 11]
```diff
@@ -0,0 +1,7 @@
+export default function LoginPage() {
```
Comment (ioayoub) [inline frontend/src/pages/User/LoginPage.jsx:1]: Empty pages are not allowed

**Review by ioayoub — CHANGES_REQUESTED** [review]

**Pull request closed without merging**